invenio_communities/communities/schema.py

```diff
@@ -76,6 +76,13 @@ def is_not_uuid(value):
         pass
 
 
+def is_not_reserved(value):
+    """Make sure value is not a reserved keyword."""
+    return validate.NoneOf(
+        iterable=["new", "search"], error=("The ID cannot be a reserved keyword.")
```

Comment:
question: couldn't we parse the communities routes config to dynamically identify these?